packages/parser/src/schemas/transfer-family.ts

```diff
@@ -0,0 +1,25 @@
+import { z } from 'zod';
+
+/**
+ * TransferFamilySchema validates events coming from AWS Transfer Family.
+ */
+export const TransferFamilySchema = z.object({
```

Comment:
Please add the export [here](https://github.com/aws-powertools/powertools-lambda-typescript/blob/66364b5f9c74df7eefc695aab788c1f4008e7250/packages/parser/package.json#L111-L114).

Reply:
Also, export at the end of the file rather than the specific schema like [here](https://github.com/aws-powertools/powertools-lambda-typescript/blob/66364b5f9c74df7eefc695aab788c1f4008e7250/packages/parser/src/schemas/cloudwatch.ts#L86).